manifests/redis_exporter.pp

```diff
@@ -95,7 +95,7 @@
   String $package_name,
   String $user,
   String $version,
-  String $config_file            = undef,
+  String $config_file            = '',
```

Comment:
Why switch from undef to empty string? Leave it as undef and instead of String use Optional[Stdlib::Absolutepath]

Reply:
Hi @tuxmea , I'm just trying to make Trevis happy. Trevis failed on the previous PR because of it and I'm trying to use your own code as guide line but no go. Before submit my first PR, I've looked SNMP exporter and Blackbox and both has a "config_file" String parameter without a default value, but here Trevis fails.
Well, the code is working great at my machine, one day when I had patience to deal with this Trevis again I will submit a new PR.

Regards.